### PR TITLE
Add MP4 no lab spec

### PR DESCRIPTION
**Pull request by andrewipark**
DONT MERGE UNTIL PACKAGES FIXED

### CODE REVIEW

File: _assignments/mp4_nolab.md [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+points: 150
+---
+
+## NO VR
+
+The VR lab is closed, so this will be a regular Unity project with no VR additions.
+
+## Overview
+
+In this MP, you will build a flight simulator in Unity. You must make a start menu, tutorial, script to support your plane, make a UI, and optimize the game to fulfill requirements from the [Oculus Best Practices Guide](https://developer.oculus.com/design/latest/concepts/book-bp/)!
```
Comment (zmabry2) [inline _assignments/mp4_nolab.md]: How are we expecting them to fulfill Oculus Best Practices, I think we should remove this or explain further

File: _assignments/mp4_nolab.md [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+Now, scatter some spheres in the sky.
+When your plane successfully shoots a sphere, the sphere should disappear for 5 seconds, and then reappear.
+Make sure the spheres are easily visible.
+Successfully shooting the spheres should increment the game score, which we will talk about in the next section.
+
+## UI
+
+You must implement some controls that will form the user interface for your game.
+Remember to consult the Oculus Best Practices Guide and share ideas with your teammates.
+If you are still confused, think about the games you have played before, and check [this](https://developer.oculus.com/blog/introducing-the-oculus-sample-framework-for-unity-5/).
```
Comment (zmabry2) [inline _assignments/mp4_nolab.md]: see above comment

File: _assignments/mp4_nolab.md [hunk 3]
```diff
@@ -0,0 +1,162 @@
+---
+layout: assignment
+title: "MP4: Flight Simulator"
+index: 41
+due: Apr. 13, 2020 @ 11:59 PM
+material:
+  -
+    name: MP4 Unity Package
+    url: /mps/MP4.unitypackage
```
Comment (RizkyWellyanto) [inline _assignments/mp4_nolab.md]: @andrewipark actually, the new unity package is >100MB. Maybe we should just put a link to the GDrive?

Link: https://drive.google.com/open?id=1jmZiOKYN9swOgJSkfBTdUmYnqHvCtN55

File: _assignments/mp4_nolab.md (hunk 1 above)
Comment (andrewipark) [reply _assignments/mp4_nolab.md:70]: fixed

File: _assignments/mp4_nolab.md (hunk 2 above)
Comment (andrewipark) [reply _assignments/mp4_nolab.md:124]: fie

File: _assignments/mp4_nolab.md (hunk 3 above)
Comment (andrewipark) [reply _assignments/mp4_nolab.md]: fixed now

**Pull request merged**